HarClient.Tests/HarSerialisationBehaviour.cs

```diff
@@ -0,0 +1,82 @@
+﻿using HarSharp;
+using JBlam.HarClient.Tests.Validation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+
+namespace JBlam.HarClient.Tests
+{
+    [TestClass]
+    public class HarSerialisationBehaviour
```

Comment:
Missing some core coverage; need tests or issues-raised for:

- validation of HARs with entries
- assert Time works; consider spec requirement that Time >= 0
- inappropriate default values in the non-optional Timings properties
- any other inappropriate default values in the HarSharp API
- production/validation of query strings
- validation of headers
- production/validation of cookies

Reply:
- HAR entries: TODO
- Time/timing: #11 
- others: depends on #12

Reply:
HAR entries depends on both #11 and #12, outside scope of this PR.